study.md

```diff
@@ -24,7 +24,10 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+A single-page application is essentially what the name implies. There is no change in
+URL, and the page does not refresh to accommodate a new rendering of the HMTL. All
+interaction with the back-end server happens through AJAX requests, and this creates
+a more fluid, responsive application.  
```

Comment:
You can swap html templates in and out, and also you can change the url sometimes based on the view state